docs/modeling/included-types.rst

```diff
@@ -0,0 +1,45 @@
+Including & Excluding Types
+===========================
+
+Including a type in the model means that EF has metadata about that type and will attempt to read and write instances from/to the database.
+
+.. contents:: In this article:
+    :depth: 3
+
+Conventions
+-----------
+
+By convention, types that are exposed in ``DbSet`` properties on your context are included in your model. In addition, types that are mentioned in the ``OnModelCreating`` method are also included. Finally, any types that are found by recursively exploring the navigation properties of discovered types are also included in the model.
+
+For example, in the following code listing all three types are discovered:
+ - ``Blog`` because it is exposed in a ``DbSet`` property on the context
```

Comment:
That won't include a type in the model since that API is only used after the model is already built.